plenum/server/view_change/view_changer.py

```diff
@@ -497,54 +473,16 @@ def start_view_change(self, proposed_view_no: int, continue_vc=False):
         self.provider.notify_view_change_start()
         self.provider.start_view_change(proposed_view_no)
 
-    def _start_selection(self):
-
-        error = None
-
-        if not self.provider.is_node_synced():
-            error = "mode is {}".format(self.provider.node_mode())
-        elif not self.has_acceptable_view_change_quorum:
-            error = "has no view change quorum or no message from next primary"
-        else:
-            rv = self.get_sufficient_same_view_change_done_messages()
-            if rv is None:
-                error = "there are not sufficient same ViewChangeDone messages"
-            elif not self._verify_primary(*rv):
-                error = "failed to verify primary"
-
-        if error is not None:
-            logger.info('{} cannot start primary selection because {}'.format(self, error))
-            return
-
-        if self.is_behind_for_view:
-            logger.info('{} is synced and has an acceptable view change quorum '
-                        'but is behind the accepted state'.format(self))
-            self.provider.start_catchup()
-            return
-
-        self.provider.select_primaries()
-
-        # if self.view_change_in_progress:
-        #     self.view_change_in_progress = False
-        #     self.provider.notify_view_change_complete()
-        #     # when we had INSTANCE_CHANGE message, they added into instanceChanges
-        #     # by msg.view_no. When view change was occured and view_no is changed,
-        #     # then we should delete all INSTANCE_CHANGE messages with current (already changed)
-        #     # view_no (which used in corresponded INSTANCE_CHANGE messages)
-        #     # Therefore we delete all INSTANCE_CHANGE messages from previous and current view number
-        #     self.instance_changes.remove_view(self.view_no)
-        #     self.previous_view_no = None
-        #     self.previous_master_primary = None
+    # TODO: Check whether these still need to be called somewhere after view change:
+    #  - self.provider.select_primaries()
+    #  - self.provider.notify_view_change_complete()
+    #  - self.instance_changes.remove_view(self.view_no)
```

Comment:
This is already called by new code in `_process_new_view_accerted` (BTW there is a typo there)